.github/workflows/auto-update-versions.yml

```diff
@@ -24,7 +24,9 @@ jobs:
           .github/workflows/scripts/auto-update-version.sh "opentelemetry-collector-releases" "collectorVersion" "content/en/docs/collector/_index.md"
           git reset --hard origin/main
           .github/workflows/scripts/auto-update-version.sh "opentelemetry-java" "javaVersion" "content/en/docs/instrumentation/java/_index.md"
+          git reset --hard origin/main
           .github/workflows/scripts/auto-update-version.sh "opentelemetry-java-instrumentation" "javaInstrumentationVersion" "content/en/docs/instrumentation/java/automatic/annotations.md"
+          git reset --hard origin/main
```

Comment:
This reset is a noop from the perspective of the update job, but it makes it less likely that someone will forget.